02 - Community Expansions/02.05 All Community Expansions/Plugins/graph-analysis.md

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
I don't know if it matters at all, but I compared this with the current output generated by `update-releases.py` - and these were the differences...

![image](https://user-images.githubusercontent.com/4840096/139823338-785ca5dd-c2d6-4022-9b1d-05d66a2b5392.png)

Reply:
I think it should be fine to accept this PR and then rerun the script and merge in the changes...

Reply:
Cool cool, I am happy with those changes after merging :)

Reply:
@SkepticMystic Could you pull the latest changes from main? Just to make sure we're not missing anything.

Reply:
Cool cool, just pulled :)